Dockerfile

```diff
@@ -94,7 +94,10 @@ COPY --from=assets-precompile /app /app
 COPY --from=assets-precompile /usr/local/bundle/ /usr/local/bundle/
 COPY --from=middleman /public/ /app/public/
 
-RUN echo "cd /app && /usr/local/bin/bundle exec rails c" > /root/.ash_history
+RUN echo export PATH=/usr/local/bin:\$PATH > /root/.ashrc
```

Comment:
```suggestion
RUN echo export PATH=/usr/local/bin:/usr/local/bundle/bin:\$PATH > /root/.ashrc
```

Reply:
Could also add an `export BUNDLE_GEMFILE=/app/Gemfile` somewhere for extra safety

Reply:
discussed in person: will move to that in the next step for now going to try this initially